src/blog/lagom-1-5-0-M1.md

```diff
@@ -1,27 +1,47 @@
+
+### Akka, Play, Lagom cross-product collaboration
+
+The promotion of `Akka Coordinated Shutdown` mentioned above is not exceptional. We are laying groundwork for integration with [Akka gRPC](https://github.com/akka/akka-grpc) to add it as a transport alternative, allowing seamless communication between applications written using Akka, Play, Lagom and others using gRCP.
+
+This cross-product collaboration is the first of many more to come. The motivation is to allow users developers to more naturally move between Akka, Play and Lagom.
+
+[Akka's Service Discovery](https://developer.lightbend.com/docs/akka-management/current/discovery.html) will become a building block of Lagom's `ServiceLocator` and help standardize service discovery scenarios. This will simplify production settings to tune cluster bootstrap and other service discovery strategies and combine them. Lagom `1.5.0-M2` started the refactor process to move to an implementation that's based 100% on Akka's Service Discovery. This new library provides an extensible API for looking up service addresses by name from within Akka-based services, including integration with Kubernetes, Marathon, Consul, and others. It is similar to Lagom's ServiceLocator API, but can be used in any Akka or Play application, as well as in Lagom. It is a foundational building block for many of the upcoming cross-product features, including gRPC support, and with this release it works seamlessly in Lagom's runAll development mode without additional configuration. We are planning to integrate this more fully with Lagom in future 1.5.0 milestones and beyond.
+
+### Other improvements
+
+- Updates Alpakka Kafka to version [0.22](https://github.com/akka/alpakka-kafka/releases). This also includes a version upgrade of the Kafka Server embedded in Development Mode to `1.0.1`.
+
+- Improves Development Mode settings so each service can choose what network interface to bind to (the new default for all services is `127.0.0.1`), which is particularly useful in some OS or corporate environments. Every service run in Developer Mode has slightly different settings, see the docs for [Service Gateway](/documentation/latest/java/ServiceLocator.html#Default-address1), [Service Locator](/documentation/latest/java/ServiceLocator.html#Default-address) or [your services](/documentation/latest/java/ConfiguringServicesInDevelopment.html#How-are-addresses-bound-by-services?).
```

Comment:
Maybe we can expand this:

> Improves Development Mode settings so each service can choose what network interface to bind to, which is particularly useful in some OS or corporate environments where VPNs or proxies can interfere with inter-service communication. The new default for all services is to listen only on the loopback address `127.0.0.1` by default, which has better compatibility, but disallows communication from other hosts on the network. This can be configured to use another address if you need to share a Lagom development environment on a local network. Every service run in Developer Mode has slightly different settings, see the docs for [Service Gateway](/documentation/latest/java/ServiceLocator.html#Default-address1), [Service Locator](/documentation/latest/java/ServiceLocator.html#Default-address) or [your services](/documentation/latest/java/ConfiguringServicesInDevelopment.html#How-are-addresses-bound-by-services?).

As an aside, this also makes me wonder whether there should be a configurable default address for all services, so you don't have to set it in so many different places.